src/components/InputPhone/index.tsx

```diff
@@ -62,18 +63,25 @@ const InputPhone: React.FC<Props> = props => {
     const selectCountry = (country: CountryType) => {
         setCountry(country);
         setIsOpen(false);
+        onChange({
+            countryCode: selectedCountry.countryCode,
+            isoCode: selectedCountry.isoCode,
+            phone: phoneValue,
+        });
     };
 
-    const handleChange = (phoneValue: string) => {
+    const handleChange = (phoneNumberValue: string = '') => {
+        setPhoneValue(phoneNumberValue);
```

Comment:
There is no need to use a state for this, Input Phone is a controlled component, then we only need one source of truth and it is the value prop.